src/sql/src/plan/func.rs

```diff
@@ -1395,6 +1395,22 @@ lazy_static! {
                         column_names,
                     })
                 })
+            },
+            "repeat" => {
+                params!(Int64) => unary_op(move |_ecx, n| {
+                    Ok(TableFuncPlan {
+                        func: TableFunc::Repeat,
+                        exprs: vec![n],
+                        column_names: vec![]
+                    })
+                }),
+                params!(Int32) => unary_op(move |_ecx, n| {
+                    Ok(TableFuncPlan {
+                        func: TableFunc::Repeat,
+                        exprs: vec![n],
+                        column_names: vec![]
+                    })
+                })
```

Comment:
Per comment above, just impl this for `Int64`. Int32s will be automatically coerced to Int64s.